sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -4,6 +4,20 @@ var srlog = SIREPO.srlog;
 var srdbg = SIREPO.srdbg;
 
 SIREPO.NUMBER_REGEXP = /^\s*(\-|\+)?(\d+|(\d*(\.\d*)))([eE][+-]?\d+)?\s*$/;
+SIREPO.SAFE_PATH_REGEXP_EXCLUDE_CHARS = [   // array of illegal filename characters
```

Comment:
We generally don't do "side comments". In this particular case, the name is clear so the comment is unnecessary.

I would say "UNSAFE_PATH_CHARS" for a name. I understand the long name, but it's overdone in this case.